src/kixi/lapita.clj

```diff
@@ -109,13 +128,24 @@
        ds/dataset))
 
 ;; Examples of uses
-(comment (def repairs-data (load-csv "data/historic-repairs-2014-2015.csv"))
+(comment (def repairs-data (csv-to-maps-coll "data/historic-repairs-2014-2015.csv"))
```

Comment:
Well, they're very handing for me. But I guess I should save those examples locally!